tests/experimental/pydantic/test_conversion.py

```diff
@@ -812,17 +812,20 @@ def test_can_convert_pydantic_type_to_strawberry_newtype():
     class User(BaseModel):
         age: int
         password: Optional[Password]
+        passwords: List[Password]
 
     @strawberry.experimental.pydantic.type(User)
     class UserType:
         age: strawberry.auto
         password: strawberry.auto
+        passwords: strawberry.auto
 
-    origin_user = User(age=1, password="abc")
+    origin_user = User(age=1, password="abc", passwords=["hunter2"])
```

Comment:
To make this more of a _unit_ test, especially as this was a regression fix, can you turn this into a separate test case?

Reply:
done!